packages/teams-js/src/private/copilot.ts

```diff
@@ -0,0 +1,54 @@
+import { ensureInitialized } from '../internal/internalAPIs';
+import { errorNotSupportedOnPlatform } from '../public/constants';
+import { runtime } from '../public/runtime';
+import { AppEligibilityInformation } from './interfaces';
+
+/**
+ * @beta
+ * @hidden
+ * Namespace to delegate M365 chat app specific APIs
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+export namespace copilot {
+  /**
+   * @beta
+   * @hidden
+   * User information required by specific apps
+   * @internal
+   * Limited to Microsoft-internal use
+   */
+  export namespace eligibility {
+    /**
+     * @hidden
+     * @internal
+     * Limited to Microsoft-internal use
+     * @beta
+     * This function is called by M365Chat app
```

Comment:
I would probably remove this. I don't think it's relevant. Same goes for line 41